profiles/models.py

```diff
@@ -1,3 +1,55 @@
 from django.db import models
+from django.core.validators import RegexValidator
 
-# Create your models here.
+from validation.validate_password import validate_password
+from validation.validate_phone_number import validate_phone_number_len, validate_phone_number_is_digit
+from validation.validate_edrpou import validate_edrpou
+from validation.validate_foundation_year import validate_foundation_year
+from validation.validate_image import validate_image_size, validate_image_format
+
+#REGIONS = ["east", "west", "north", "south"]
+REGIONS = (('E', "east"), ('W', "west"), ('N', "north"), ('S', "south"))
+
+class Profile(models.Model):
+
+    profile_id = models.IntegerField(primary_key=True)
+    comp_category = models.ManyToManyField("Category")
+    comp_activity = models.ManyToManyField("Activity")
+    comp_name = models.CharField(max_length=50)
+    comp_official_name = models.CharField(max_length=255, unique=True)
+    comp_region = models.CharField(max_length=128, choices=REGIONS)
+    comp_common_info = models.CharField(max_length=255)
+    comp_phone_number = models.CharField(
+        max_length=12,
+        validators=[
+            validate_phone_number_is_digit,
+            validate_phone_number_len,
+            RegexValidator(r"^\d+$", message="Phone number must contain only numbers."),
+        ]
+    )
+    comp_EDRPOU = models.IntegerField(unique=True, validators=[validate_edrpou])
+    comp_year_of_foundation = models.SmallIntegerField(validators=[validate_foundation_year])
+    comp_service_info = models.TextField()
+    comp_product_info = models.TextField()
+    comp_address = models.TextField()
+    comp_registered = models.BinaryField(max_length=1)
```

Comment:
what is the purpose of this attribute?

Reply:
yep

Reply:
Sorry for the possible misunderstanding, but it looks like we can use only one switch on/off in comp_startup. Although, with the help of two attributes at the same time, we can cover the requirement when the subject can be both a startup and a company at the same time. Leave it for now.

Reply:
Ok, we will think about this problem. Thank you for feedback.